config/config.go

```diff
@@ -510,6 +510,13 @@ func (conf *Config) AddHealthDefault() {
 						Direction: ".*",
 						Failure:   10,
 					},
+					{
+						Code:      "^-$", // no response is indicated with a "-" code
+						Protocol:  "http|grpc",
+						Direction: ".*",
+						Degraded:  0,
+						Failure:   10,
```

Comment:
it's ok but we don't need to set degraded to 0 (it's 0 by default) we can remove that line if you want